Groovy/Groovy.sublime-syntax

```diff
@@ -386,6 +421,32 @@ contexts:
       scope: storage.type.def.groovy
     - match: '\b(boolean|byte|char|short|int|float|long|double)(?:\[\s*\])*\b'
       scope: storage.type.primitive.groovy
+  single-dollar-string-interpolation:
+    - match: \${{single_dollar_interpolation_identifier}}
+      scope: variable.other.interpolated.groovy
+      push:
+        - match: \.(?={{single_dollar_interpolation_identifier}})
+          scope: keyword.operator.navigation.groovy
```

Comment:
I think the other syntaxes tend to scope this as `punctuation.accessor.dot` as opposed to a `keyword`.

Reply:
I'm happy to update it, I've mostly been guessing/copying when it comes to scopes; in this case I copied it from https://github.com/sublimehq/Packages/blob/ce7af4d6177d0340230893e49742070ae4143246/Groovy/Groovy.sublime-syntax#L164

Reply:
yeah, the Groovy syntax hasn't been updated for a while so probably contains a few existing scopes that are not standardized; it'd be great if you could "update" that line too, please. :) The `punctuation.accessor` scope can be found in the official scope naming documentation at https://www.sublimetext.com/docs/3/scope_naming.html.